eng/liveBuilds.targets

```diff
@@ -0,0 +1,94 @@
+<Project>
+
+  <PropertyGroup>
+    <CoreCLROSGroup Condition="'$(CoreCLROSGroup)' == ''">$(OSGroup)</CoreCLROSGroup>
+    <CoreCLRConfiguration Condition="'$(CoreCLRConfiguration)' == ''">$(Configuration)</CoreCLRConfiguration>
```

Comment:
Can I build CoreCLR for Release and then consume it in a Debug build of Libraries/installer? I presume I would need to pass these arguments down if I want to achieve that?

Reply:
Yep! Just set the `CoreCLRConfiguration` or `LibrariesConfiguration` parameters to the configuration you want and it will use that configuration instead. Due to the lack of a good customization point in Arcade, the root build script will always build with the same configuration all-up, so that's why it defaults to the same config.

Reply:
I see. It might be good if we could explore building partitions in different configurations as part of the build script. 

The reason why I mention that is because when people add an API to coreclr and then try to add tests for it in libraries, if they endup with a Debug coreclr, their tests will be super slow, so it might be good to configure it from the root script to build each partition in different configurations as one command. Rather than having to do `coreclr.cmd -Release && libraries.cmd /p:CoreCLRConfiguration=Release`.